worker/src/main/java/org/apache/celeborn/service/deploy/worker/storage/FileWriter.java

```diff
@@ -434,6 +431,19 @@ public PartitionSplitMode getSplitMode() {
     return splitMode;
   }
 
+  @Override
+  public void notifyError(String mountPoint, DiskStatus diskStatus) {
+    destroy(
```

Comment:
It will destroy all filewriters on a disk if one flush reports, error, is this what we want?

Reply:
> It will destroy all filewriters on a disk if one flush reports, error, is this what we want? cc @AngersZhuuuu

When flush error, should not report critical error, I have done this #928

Reply:
Currently ReadWriteCheck will trigger notifyError, and if it fails, it will cause all filewriters closed, is it dangerous? cc @AngersZhuuuu